includes/header.php

```diff
@@ -220,22 +174,15 @@
                 <div class="modal-body" >
                     <!-- File name will be displayed here -->
                     <div class="alert alert-danger" role="alert" id="fileModalBody"></div>
-                
-                    <hr>
-                    <div id='column_names' hidden></div>
-                    <hr>
-                    <div id='rows_values' hidden></div>
-
-                    
                 </div>
                 <div class="modal-footer">
                     <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-                    <button type="button" class="btn btn-primary" data-bs-dismiss="modal" onclick = 'confirmUpload()'>Confirm upload</button>
+                    <button type="button" class="btn btn-primary" data-bs-dismiss="modal" onclick = 'confirmUpload("", "")'>Confirm upload</button>

```

Comment:
`uploader.js`, where `confirmUpload` is defined (and apparently has changed) is not part of this PR. Is this a fix for something that broke during refactoring? Or should more files be included in this PR?